etl3.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I totally get why you did this approach and it's probably quicker than having a panda doing the load but it isn't very translatable into the aws pipeline. 

I mean I think it's fine to merge but we need a follow up ticket to make this a script. You can pretty much rip out components of ETL1 cos it does similar stuff and use them in this.